apps/web/pages/auth/logout.tsx

```diff
@@ -29,11 +30,17 @@ export function Logout(props: Props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [props.query?.survey]);
   useEffect(() => {
-    if (props.query?.changeEmail === "true") {
-      setChangeEmail(true);
+    if (props.query?.emailChange === "true") {
```

Comment:
you can actually use the query instead of useState here

Reply:
Thanks for the feedback! i thought of the same but i thought why change it and followed the existing pattern. Will keep that in mind.